site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,455 @@ aliases: "/docs/5.1/components/orange-footer/"
+    </ul>
+  </div>
+
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Navigation -->
+  <div class="navbar-dark footer-nav">
+    <div class="container d-none d-md-flex">
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="d-md-none accordion" id="accordionExampleSmall">
+      <div class="border-bottom border-1 border-dark">
+        <h3 class="container accordion-header" id="headingOneSmall">
```

Comment:
Is the container class usefull here?

Reply:
Yes, I think so, in order to be aligned with the other sub-components.